src/iOS/toga_iOS/widgets/imageview.py

```diff
@@ -19,6 +20,9 @@ def startup(self):
         self._impl.setTranslatesAutoresizingMaskIntoConstraints_(False)
         self._impl.setAutoresizesSubviews_(False)
 
+        # Add the layout constraints
```

Comment:
Apologies - I missed this in the first review because it wasn't strictly part of your patch; but if you're re-enabling image views, it's in your court :-)

`startup()` is an older bootstrap method for widgets; newer widgets use create(). The ImageViewInterface also needs to be updated - as it currently stands, it's a mostly empty shell. These both need to be updated before we can merge.